RETL source doc updates (#1022)
rudderlabs/rudderstack-docs: docs/sources/reverse-etl/amazon-redshift.mdx

<img src="../../assets/warehouse-actions-sources/destination.png" alt="Add destination in RudderStack" />

<div class="infoBlock">
If you have already configured a destination in RudderStack, choose the <strong>Use Existing Destination</strong> option which will take you to the <strong>Schema</strong> tab in the source settings. To add a new destination from scratch, select the <strong>Create New Destination</strong> option which will take you to the destination configuration page.
</div>

## Mapping warehouse data to destination

> After selecting a destination and configuring its connection settings, you can choose how to map the data from your Reverse ETL source to the destination using one of the following methods:

Reviewer: ```suggestion
After adding a destination for your Redshift source, you can choose how to send the data to it using any one of the following methods:
```

> - <Link to="/sources/reverse-etl/common-settings/importing-data-using-tables/">JSON mapping</Link>: It is the default method where you can choose the warehouse column and the corresponding JSON trait key from the destination to be mapped together. The below image shows the JSON mapping settings if you choose to sync data via the Table option:

Reviewer: ```suggestion
- <Link to="/sources/reverse-etl/common-settings/importing-data-using-tables/">JSON mapping</Link>: This is the default method where you can map a warehouse column to a specific JSON trait key from the destination. 

The below image shows the JSON mapping settings if you choose to sync data via the **Table** option:
```

> <img src="../../assets/warehouse-actions-sources/json-mapping.png" alt="JSON Mapping" />

Reviewer: Let's do away with the arrow or keep the arrow length consistent in both the images. WDYT?

> - <Link to="/sources/reverse-etl/features/visual-data-mapper/">Visual Data Mapper (VDM)</Link>: RudderStack supports this method only for a few selective <Link to="/sources/reverse-etl/features/visual-data-mapper/#supported-destinations">VDM-supported destinations</Link>. It offers an intuitive UI to map your warehouse columns to the specific destination fields. The below image shows the VDM settings if you choose to sync data via the Table option:

Reviewer: ```suggestion
- <Link to="/sources/reverse-etl/features/visual-data-mapper/">Visual Data Mapper (VDM)</Link>: RudderStack supports this method only for <Link to="/sources/reverse-etl/features/visual-data-mapper/#supported-destinations">specific destinations</Link>. It offers an intuitive UI to map your warehouse columns to the specific destination fields. 

The below image shows the VDM settings to sync data via the **Table** option:
```

<img src="../../assets/warehouse-actions-sources/vdm-mapping.png" alt="VDM Mapping" />

> <div class="infoBlock">

Reviewer: ```suggestion
<div class="successBlock">
```

> Note that the data sent to the destination is same in both the cases and is independent of the method chosen from above.

Reviewer: Short and crisp, thoughts?

</div>

> Further, refer to the <Link to="">JSON mapping</Link> or <Link to="/sources/reverse-etl/features/visual-data-mapper/">Visual Data Mapper</Link> guide for further steps on specifying the data to be imported (using Table or Model) and establishing a Reverse ETL connection successfully.

Reviewer: This should be a callout, IMO.
